pkg/apis/core/helper/helpers.go

```diff
@@ -371,6 +371,17 @@ func GetShootCARotationPhase(credentials *core.ShootCredentials) core.ShootCrede
 	return ""
 }
 
+// GetShootServiceAccountKeyRotationPhase returns the specified shoot service account key rotation phase or an empty
+// string.
+func GetShootServiceAccountKeyRotationPhase(credentials *core.ShootCredentials) core.ShootCredentialsRotationPhase {
+	if credentials != nil &&
+		credentials.Rotation != nil &&
+		credentials.Rotation.ServiceAccountKey != nil {
```

Comment:
Nit: for consistency reasons with the v1beta1 helper func

```suggestion
	if credentials != nil && credentials.Rotation != nil && credentials.Rotation.ServiceAccountKey != nil 
```